sessions/react-state/demo-food-order-end/src/App.js

```diff
@@ -11,19 +11,65 @@ export default function App() {
 }
 
 function FoodOrder() {
-  const [food, setFood] = useState("❓");
+  const [food, setFood] = useState("?");
   const [vegan, setVegan] = useState(true);
 
   return (
     <>
       <div>
-        <button onClick={() => setFood("🍔")}>🍔</button>
-        <button onClick={() => setFood("🍕")}>🍕</button>
-        <button onClick={() => setFood("🌮")}>🌮</button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A hamburger">
+                🍔
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A hamburger">
+            🍔
+          </span>
+        </button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A pizza">
+                🍕
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A pizza">
+            🍕
+          </span>
+        </button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A taco">
+                🌮
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A taco">
+            🌮
+          </span>
+        </button>
```

Comment:
This would store a React element / JSX to the state. We should not to this. I'd prefer to just ignore this case. The linter would not detect the rendered emoji anyway.

```suggestion
        <button onClick={() => setFood("🍔")}>
          <span role="img" aria-label="A hamburger">
            🍔
          </span>
        </button>
        <button onClick={() => setFood("🍕")}>
          <span role="img" aria-label="A pizza">
            🍕
          </span>
        </button>
        <button onClick={() => setFood("🌮")}>
          <span role="img" aria-label="A taco">
            🌮
          </span>
        </button>
```